### ISSUE TITLE
Does not work for 1.19

### ISSUE THREAD

**Issue by Artzert2215** [AlonsoAliaga/BetterRevive#7]
**Describe the bug**
Error is thrown when trying to revive in 1.19.

**To Reproduce**
1. Download latest purpur 1.19
2. Download better revive
3. Try to revive downed player

**Expected behavior**
No error.

**Logs**
```
[17:52:17 ERROR]: Could not pass event PlayerInteractAtEntityEvent to BetterRevive v0.9.1-BETA
java.lang.NoSuchMethodError: 'com.massivecraft.factions.struct.Relation com.massivecraft.factions.FPlayer.getRelationTo(com.massivecraft.factions.iface.RelationParticipator)'
        at com.alonsoaliaga.betterrevive.factions.SaberFactions.isFriendlyFaction(SaberFactions.java:23) ~[BetterRevive_0.9.1.jar:?]
        at com.alonsoaliaga.betterrevive.listeners.InteractAtEntityListener.onInteractAtEntity(InteractAtEntityListener.java:58) ~[BetterRevive_0.9.1.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor528.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:75) ~[purpur-api-1.19-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[purpur-api-1.19-R0.1-SNAPSHOT.jar:git-Purpur-1670]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[purpur-api-1.19-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:666) ~[purpur-api-1.19-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl$6.performInteraction(ServerGamePacketListenerImpl.java:2746) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl$6.a(ServerGamePacketListenerImpl.java:2803) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundInteractPacket$InteractionAtLocationAction.dispatch(ServerboundInteractPacket.java:161) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundInteractPacket.dispatch(ServerboundInteractPacket.java:80) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleInteract(ServerGamePacketListenerImpl.java:2738) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundInteractPacket.handle(ServerboundInteractPacket.java:67) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundInteractPacket.handle(ServerboundInteractPacket.java:12) ~[?:?]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:51) ~[?:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[purpur-1.19.jar:git-Purpur-1670]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1385) ~[purpur-1.19.jar:git-Purpur-1670]
        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:183) ~[purpur-1.19.jar:git-Purpur-1670]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1362) ~[purpur-1.19.jar:git-Purpur-1670]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1355) ~[purpur-1.19.jar:git-Purpur-1670]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1333) ~[purpur-1.19.jar:git-Purpur-1670]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1218) ~[purpur-1.19.jar:git-Purpur-1670]
        at net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:307) ~[purpur-1.19.jar:git-Purpur-1670]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
```
**Important (please complete the following information):**
 - Java version: 17
 - Plugin version: 0.9.1-BETA
 - Server version purpur-1666

**Additional context**
none

**Comment by AlonsoAliaga** [comment]
They probably changed methods in the main plugin.
Please provide some more information. Run `/version Factions` to display plugin information.
You should be using latest version, when possible join our server on [discord](https://alonsoaliaga.com/discord) to be able to reply a bit faster.

If you don't need factions hook, disable Factions hook for now in config.yml file

**Comment by Artzert2215** [comment]
/version factions return:
```
[21:41:04 INFO]: Factions version 1.6.9.5-U0.6.12-b289
[21:41:04 INFO]: Website: https://www.spigotmc.org/resources/factionsuuid.1035/
[21:41:04 INFO]: Authors: Olof Larsson, Brett Flannigan, drtshock, dariasc, CmdrKittens and mbaxter
```

**Comment by AlonsoAliaga** [comment]
I understand now, please, disable Factions hook. I don't support FactionsUUID. However, since plugin name is also "Factions", BetterRevive plugin is thinking it is SaberFactions (which is the one supported). Sorry for the inconvenience, I'll add more checks to make sure it's the right Factions plugin; for now, just disable Factions hook in config.yml. That factions plugin is not supported

**Comment by Artzert2215** [comment]
Thanks!

**Comment by AlonsoAliaga** [comment]
No problem! Closing ticket.
(Fixed in 0.9.2-BETA pending to release)
